### ISSUE TITLE
Input normalisation for MViT

### ISSUE THREAD

**Issue by LemurPwned** [facebookresearch/pytorchvideo#131]
As I can't see the model on TorchHub (at least at the date of writing it), I don't know what would be the correct input normalisation for the provided pretrained MViT models.

Do you have any info on the mean and std for image normalisation required for that pretrained model? Is it simply [0.5, 0.5, 0.5] for both mean and std?

I also couldn't find any clues in the original repo: https://github.com/facebookresearch/SlowFast/tree/master/configs/Kinetics
